src/components/annual-report/section1/section1Calculations.js

```diff
@@ -107,27 +107,31 @@ const getRegionsWithLowScoreCompletenessOfIndicator = (
     // TODO: variables naming in this function could be improved
     const regionsWithLowScore = []
     indicatorObjects.forEach((object) => {
-        const correspondingDatasetID = object.correspondingDatasetID
+        const correspondingDatasetIDs = object.correspondingDatasetIDs
         const ou = object.ou
 
-        if (expectedReportsByLevel[correspondingDatasetID]) {
+        let subOrgUnitScore = 0
+
+        for (const correspondingDatasetID of correspondingDatasetIDs) {
+            if (!expectedReportsByLevel[correspondingDatasetID]) {
+                console.log(
+                    `No dataset found for ID: ${correspondingDatasetID}`
+                )
+            }
             const matchingDataset = expectedReportsByLevel[
                 correspondingDatasetID
             ].find((dataset) => dataset.ou === ou)
             if (matchingDataset) {
-                const score = matchingDataset.score
-                const reportingPercentage = (object.actualValues / score) * 100
-
-                if (reportingPercentage < object.threshold) {
-                    regionsWithLowScore.push(object.orgUnitLevelsOrGroups)
-                }
+                subOrgUnitScore += matchingDataset.score ?? 0
             } else {
-                console.log(
-                    `No matching dataset found for Region: ${object.orgUnitLevelsOrGroups}`
-                )
+                ;`No matching dataset ${correspondingDatasetID} found for SubOrgUnit: ${object.orgUnitLevelsOrGroups}`
             }
-        } else {
-            console.log(`No dataset found for ID: ${correspondingDatasetID}`)
+        }
+        const reportingPercentage =
+            (object.actualValues / subOrgUnitScore) * 100
```

Comment:
this score is initialized to zero and looking at line 125 it could stay equals to zero. not alarming now but i think we are risking dividing by zero.

Reply:
Yes. That's a good point. I think ideally we would do a review of the calculations for edge cases and implement consistent handling for this (e.g. flagging as an invalid value if dividing by zero). I think this work would be better suited as part of another ticket